app/javascript/components/admin/AuditModal.js

```diff
@@ -257,6 +258,25 @@ class AuditModal extends React.Component {
             <b>User Signed In</b>
           </span>
         );
+      case 'manual_lock_reason':
+        // Generic audit message in case before & after values not provided
+        if (!change.details || !Array.isArray(change.details) || !change.details.length) {
+          return (
+            <span>
+              <b>Manually set Status</b>
+              <InfoTooltip tooltipTextKey={'manualLockReasonAudit'} location="right"></InfoTooltip>: Updated
+            </span>
+          );
+          // More detailed audit message when before & after values provided
+        } else {
+          return (
+            <span>
+              <b>Manually set Status</b>
+              <InfoTooltip tooltipTextKey={'manualLockReasonAudit'} location="right"></InfoTooltip>: Changed from &quot;{change.details[0]}&quot; to &quot;
```

Comment:
I don't love how the tooltip interrupts the flow of reading this.  Since the table is essentially two lines tall, thoughts on doing something like this?

<img width="1104" alt="Screen Shot 2022-02-24 at 3 19 06 PM" src="https://user-images.githubusercontent.com/35042815/155601127-314ed644-c485-4a3a-86ab-1b5cd03adff2.png">

Reply:
The other action options would also probably have to change for consistency

Reply:
Having all the Action entries be 2-lined in this format looks good to me. Will work on it.

Reply:
Addressed: [2a0e541](https://github.com/SaraAlert/SaraAlert/pull/1222/commits/2a0e541e181900b4c5d4bdb904fffd1a53ef229b)